partcad/src/partcad/wrappers/wrapper_render_obj.py

```
@@ -36,8 +35,7 @@ def process(path, request):
         with open(path, "w", encoding="utf-8", buffering=256 * 1024) as f:
             f.write("# OBJ file\n")
             for v in vertices:
-                f.write("v %.4f %.4f %.4f\n" % (v.x, v.y, v.z))
-                # f.write("v %.4f %.4f %.4f\n" % (v[0], v[1], v[2]))
+                f.write("v %.4f %.4f %.4f\n" % (v[0], v[1], v[2]))
```

Ruff (0.8.2)

Use format specifiers instead of percent format

Replace with format specifiers

(UP031)